.github/workflows/publish-pacts.yaml

```diff
@@ -22,9 +22,7 @@ jobs:
           yarn install
 
       - name: Run tests
-        run: yarn test Cbas.test.js # As of now, this is the only contract test file in terra-ui.
-                                    # When another test file is added, a naming convention for contract tests should be established,
-                                    # so that contract tests can be run in isolation.
+        run: yarn test CbasPact.test.js TermsOfServicePact.test.ts
```

Comment:
I don't mind these being listed manually for right now, but could this be changed to something that catches future pact tests that follow the naming convention? so that this list doesn't need to be updated each time a new test is added.

I don't feel very strongly about it, but that is what I had in mind when I suggested the naming convention

Reply:
Ah right that makes sense, i can look into that, maybe we can use a regex like `\b\w*Pact\w*\.test\.(js|ts)\b`